unstructured/partition/lang.py

```diff
@@ -234,7 +234,14 @@ def tesseract_to_paddle_language(tesseract_language: str = "eng") -> str:
     :return: str, corresponding language code for PaddleOCR or None if not found
     """
 
-    return PYTESSERACT_TO_PADDLE_LANG_CODE_MAP.get(tesseract_language, tesseract_language)
+    lang = PYTESSERACT_TO_PADDLE_LANG_CODE_MAP.get(tesseract_language)
+    if not lang:
+        logger.warning(
+            f"{lang} is not a language supported by PaddleOCR, proceed with `en` instead."
```

Comment:
As a reader would read this message, are we requesting an action from them or reporting what the system is doing?

If we need the user to fill in a parameter with "en" or something maybe we can spell that out a bit better.

But if we're just telling them we've substituted "en" then "proceeding" would be the right verb form.

Also, if `not lang` then lang will be `""` or `None`, so the message will read " is not a language supported..." or "None is not a language supported ...". Do we want to use `tesseract_language` instead or some other requested value?